Mathlib/Analysis/SpecialFunctions/MulExpNegSq.lean

```diff
@@ -0,0 +1,193 @@
+/-
+Copyright (c) 2024 Jakob Stiefel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jakob Stiefel
+-/
+import Mathlib.Data.Complex.ExponentialBounds
+import Mathlib.Topology.ContinuousMap.Bounded.Basic
+
+/-!
+# Definition of `mulExpNegSq`, `mulExpNegMulSq` and properties
+
+The mapping mulExpNegMulSq transforms a function `g` into another function
+`mulExpNegMulSq g ε : fun x => g x * Real.exp (-ε * g x * g x)`.
+
+## Main Properties
+
+- `abs_mulExpNegMulSq_le`: For fixed `ε > 0`, the mapping `mulExpNegMulSq g ε` is bounded by
+  `Real.sqrt ε⁻¹` uniformly in `g`;
+- `tendsto_mulExpNegMulSq`: For fixed function `g`, the mapping `mulExpNegMulSq g ε`
+  converges pointwise to `g` as `ε → 0`;
+- `continuous_mulExpNegMulSq`: If the function `g` is continuous, the function `mulExpNegMulSq g ε`
+  is continuous;
+- `abs_mulExpNegMulSq_le_norm`: For a fixed bounded continuous function `g`, the mapping
+  `mulExpNegMulSq g ε` is bounded by `norm g`, uniformly in `ε ≥ 0`;
+-/
+
+open NNReal ENNReal BoundedContinuousFunction Filter
+
+namespace Real
+
+section mulExpNegSq
+
+/-! ### Definition and properties of `fun x => x * Real.exp (- x * x)`-/
+
+/-- Definition of `mulExpNegSq` -/
+noncomputable
+def mulExpNegSq (x : ℝ) := x * exp (- x * x)
+
+theorem mulExpNegSq_apply (x : ℝ) : mulExpNegSq x = x * exp (- x * x) :=  rfl
+
+theorem neg_mulExpNegSq_neg (x : ℝ) : - mulExpNegSq (-x) = mulExpNegSq x  := by
+  simp only [mulExpNegSq, neg_mul, neg_neg, mul_neg]
```

Comment:
According to recent discussions, there seems to be a consensus emerging that a terminal simp is more maintainable if it's not squeezed. (I hope I unsqueezed it correctly: I did not test that code)
```suggestion
  simp [mulExpNegSq]
```